spec/index.bs

```diff
@@ -4238,20 +4238,25 @@ dictionary GPUBlendComponent {
 enum GPUBlendFactor {
     "zero",
     "one",
-    "src-color",
-    "one-minus-src-color",
-    "src-alpha",
-    "one-minus-src-alpha",
-    "dst-color",
-    "one-minus-dst-color",
+    "src0-componentwise",
+    "one-minus-src0-componentwise",
+    "src0-alpha",
+    "one-minus-src0-alpha",
+    "dst-componentwise",
+    "one-minus-dst-componentwise",
     "dst-alpha",
     "one-minus-dst-alpha",
-    "src-alpha-saturated",
-    "blend-color",
-    "one-minus-blend-color"
+    "src0-alpha-saturated",
+    "blendcolor-componentwise",
+    "one-minus-blendcolor-componentwise"
 };
 </script>
 
+Issue: {{GPUBlendOperation}}s "min" and "max" actually ignore the {{GPUBlendFactor}}.
+Should we validate the `srcFactor`/`dstFactor` both must be "one" with "min"/"max"?
```

Comment:
I believe validating this would be good for us. "one" makes sense there, since it can be interpret as `min(1 * Src, 1 * Dst)` correctly. But other factors are not supported, and we shouldn't pretend they are ok.